composer/callbacks/export_for_inference.py

```diff
@@ -9,6 +9,8 @@
 from copy import deepcopy
 from typing import Any, Optional, Sequence, Union
 
+from torch.nn import Module
```

Comment:
nit: standard way of importing this is import torch.nn as nn and then use nn.Module. Change here and other places you use it.

Reply:
Done!